### PR TITLE
Fix Session Fixtures Without Id

### PR DESCRIPTION
**Pull request by SEhrm**
If a test uses only fixtures without ids -- that cannot happen if the `"mpi_ranks"` fixture is used -- `_get_identifier` previously raised an `AttributeError` because `callspec` was not found.

### CODE REVIEW

**Comment by sebasgo** [comment]
Looks good. Can you please one bullet point summarizing the fix into `CHANGES.rst` under the (not yet existing) Version 0.2 headline?

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+Version 0.2
+-----------
+
+- https://github.com/dlr-sp/pytest-isolate-mpi/pull/14 fixes an edge case when using session scoped fixtures on non-parametrized tests.
```
Comment (sebasgo) [inline CHANGES.rst:7]:
```suggestion
- An unhandled edge case when using a session-scoped fixture in
  non-parametrized tests was fixed. (`#14`_)

.. _#14:  https://github.com/dlr-sp/pytest-isolate-mpi/pull/14
```
